_content/articles/elephant-good-to-think_ohnuma-reiko.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "An elephant good to think: The Buddha in Pārileyyaka forest"
+authors:
+  - "Reiko Ohnuma"
+external_url: "https://www.academia.edu/10975388/An_Elephant_Good_to_Think_The_Buddha_in_P%C4%81rileyyaka_Forest"
+drive_links:
+  - "https://drive.google.com/file/d/19uGBmWI7h-6hbJFca0rA3pbvgUYydKLc/view?usp=sharing"
+course: animals
+tags:
+  - imagery
+  - indian
+  - literature
```

Comment:
`hermeneutics` and `pali-commentaries` more specifically instead of literature. And add `buddha`